packages/y/yubikey-manager/package.yml

```diff
@@ -34,9 +28,25 @@ rundeps    :
     - python-pyscard
 build      : |
     %python3_setup
+    gzip man/ykman.1
```

Comment:
`ypkg` compresses manpages automatically at the end of the build, you don't have to do it manually.

Reply:
Thanks, was not aware of this feature.
Updated the file it accordingly and re-tested.